src/plex/plex.js

```diff
@@ -0,0 +1,56 @@
+const axios = require('axios');
+const chalk = require('chalk');
+const stripPath = require('strip-path');
+const path = require('path');
+
+class Plex {
+    name = "Plex"
+
+    constructor(publicUrl, apiKey, stripPath) {
+        const baseUrl = new URL(publicUrl);
+        this._requester = axios.create({
+            baseURL: baseUrl.toString(),
+            timeout: 20000,
+            headers: {
+                'X-Plex-Token': apiKey,
+                'Accept': 'application/json'
+            }
+        });
+        this._stripPath = stripPath;
+    }
+
+    async ping() {
+        await this._requester.get('/');
+    }
+
+    async listMovies() {
+        console.log(chalk.green('Plex'), `Searching for movies`);
+        const { data: { MediaContainer: items } } = await this._requester.get('/library/sections/3/all');
```

Comment:
This "3" (in the path) is linked to your own plex instance, you need to use `/library/sections` in order to get the movie library ids